Resources/config/services.yml

```diff
@@ -12,6 +12,21 @@ services:
         calls:
             - [setLogger, ['@logger']]
 
+    auto1.api.response.strategy.transformer:
+        class: Auto1\ServiceAPIClientBundle\Service\Response\ResponseStrategyTransformer
+        arguments:
+            $endpointRegistry: '@auto1.api.endpoint.registry'
+            $responseTransformerStrategies:
```

Comment:
you can use a tagged iterator instead of specifying each Strategy

Reply:
replaced with tagged iterator for Strategies, thanks